### ISSUE TITLE
Black suits of cards are very difficult to distinguish in Discord Dark Mode.

### ISSUE THREAD

**Issue by Drofseh** [ArtemGr/Sidekick#235]
**Describe the bug**
Title.

**To Reproduce**
1. Deal cards to get a Club or a Spade.
2. Be unable to tell which one you got.
3. Deal more cards until you get a Club and a Spade in a single hand.
4. Be unable to tell them apart.

**Expected behavior**
Black suits should be recoloured or given a border to be visible in Dark Mode.

**Screenshots**
![](https://i.gyazo.com/eab60d4d60ba01ccda04fb97221f8bde.png)
8D, JD, 2C, 5C, 5S

**Environment (please complete the following information):**
 - OS: Windows 10
 - Desktop

**Additional context**

**Comment by vagueGM** [comment]
Agreed. Adding a white border around the image should make it clear with both dark and light themes.
Maybe do all the suits for consistency and to cater for the future 'red theme'.
